C/Coin-OR/Bonmin/build_tarballs.jl

```diff
@@ -50,10 +50,6 @@ make install
 rm "${includedir}/coin-or/IpoptConfig.h"
 """
 
-
-platforms = supported_platforms()
-filter!(!Sys.isfreebsd, platforms)
-platforms = expand_cxxstring_abis(platforms)
 filter!(x -> cxxstring_abi(x) != "cxx03", platforms)
 platforms = expand_gfortran_versions(platforms)
```

Comment:
I don't think this is necessary, as far as I can see from the audit log
```suggestion
```

Reply:
Ah. We see `locks us to cxx11` but not `locks us to gfortran5`? I guess the auditor could check if the fortran versions were expanded, but it didn't see a lock and throw a warning?

Reply:
> I guess the auditor could check if the fortran versions were expanded, but it didn't see a lock and throw a warning?

That's something I'd like to be able to do, but at the moment it isn't possible: we'd need to collect all warnings (which we can't do) and check whether we built for a specific libgofrtran version and there were no warnings about it (similarly for C++ string ABI).  We need to rework the whole auditing process to be able to catch this.